config/environments/production.rb

```diff
@@ -38,14 +38,15 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
-  # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
+  config.log_formatter = proc do |severity, _time, _progname, msg|
+    "#{severity}: #{msg}\n"
+  end
 
   # Use a different logger for distributed setups.
   # require 'syslog/logger'
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
-  if ENV['RAILS_LOG_TO_STDOUT'].present?
+  if ENV['RAILS_LOG_TO_JOURNALD'].present?
     logger           = ActiveSupport::Logger.new(STDOUT)
     logger.formatter = config.log_formatter
     config.logger    = ActiveSupport::TaggedLogging.new(logger)
```

Comment:
Perhaps create a new class (like `RMT::Logger`) and move all this boilerplate logger initialization code (ENV var check, format setting, etc.) into a constructor, and then use this class everywhere?

Reply:
I think since we don't have `ActiveSupport` in the CLI (do we?) I have to make it only in context of the CLI code, right? @ikapelyukhin

Reply:
We have `ActiveSupport`: https://github.com/SUSE/rmt/blob/54170ef0d7ce5c969de8dabbf899b672db95ad70/bin/rmt-cli#L9

But you'll have to see if it is possible to just replace default Ruby logger with `ActiveSupport::Logger`.
I think it would be better to have the same logger for everything.